From a61657c848aac23c379854e59418d7cd4a8e6bf7 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Mon, 17 Oct 2022 09:00:43 +0300
Subject: [PATCH] [SW-0] package: arcboot: copy rom elf file

Currenlty, arcboot rom artifacts created as part of fw build doesn't
contain rom elf which is better to keep as part of artifacts
for final rom release. This patch add changes to copy ROM elf to
images folder which will be then picked to artifacts.

Change-Id: Ib6153c9cceed811b04ae83c9a461bfab9b3912fa
---

diff --git a/package/arcboot/arcboot.mk b/package/arcboot/arcboot.mk
index 272aeda..0f681b3 100644
--- a/package/arcboot/arcboot.mk
+++ b/package/arcboot/arcboot.mk
@@ -17,6 +17,7 @@
 ARCBOOT_INSTALL_IMAGES = YES
 define ARCBOOT_INSTALL_IMAGES_CMDS
 	$(INSTALL) -D -m 0644 $(@D)/arcboot_rom.bin $(BINARIES_DIR)/arcboot_rom.bin
+	$(INSTALL) -D -m 0644 $(@D)/arcboot_rom $(BINARIES_DIR)/arcboot_rom
 endef
 
 define HOST_ARCBOOT_INSTALL_CMDS
